hydroshare/settings.py

```diff
@@ -227,8 +230,11 @@
 # Example: "/home/media/media.lawrence.com/static/"
 STATIC_ROOT = os.path.join(PROJECT_ROOT, STATIC_URL.strip("/"))
 
-# TODO: Manifest static file storage deactivated for now, pending further debugging
-# STATICFILES_STORAGE = "hydroshare.storage.ForgivingManifestStaticFilesStorage"
+STATICFILES_STORAGE = (
+    'django.contrib.staticfiles.storage.StaticFilesStorage'
+    if TESTING
```

Comment:
Why can't we just always use the new forgiving storage calss?

Reply:
Because **collectstatic** isn’t run as part of the test setup. So if we didn't use the `StaticFilesStorage` instead of the `ForgivingManifestStaticFilesStorage` during testing, the manifest wouldn't exist and the tests might try to reference it. I guess it maybe isn't an issue for our project, because I think all of our tests passed without this logic to check whether testing...